packages/prettier-rules/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@uifabric/prettier-rules",
+  "version": "6.0.0",
+  "description": "Shared Prettier rules",
```

Comment:
Same comment - I think this should follow semver. With leaf dependencies, you'll never run into the so-called "diamond dependency" problem.

Reply:
moved to 1.0.0